src/Processor/TableOfContentProcessor.php

```diff
@@ -59,7 +63,7 @@ public function __invoke(array &$data, string $type, Content $content): void
         }
 
         $data[$this->tableOfContentProperty] = $this->generator->getTableOfContent(
-            $data[$this->contentProperty],
+            $this->crawlers->get($content, $data, $this->contentProperty, $content),
```

Comment:
```suggestion
            $this->crawlers->get($content, $data, $this->contentProperty),
```

Reply:
Since the crawler can be null here, we must early return before calling `getTableOfContent` in such case